prowler/lib/scan/scan.py

```diff
@@ -18,14 +19,23 @@ class Scan:
     _service_checks_completed: dict[str, set[str]]
     _progress: float = 0.0
     _findings: list = []
+    _mutelist_file: str = None
+    _config_file: str = None
```

Comment:
Do we need to store them in the class? I mean because they are both stored in the provider.

Reply:
True! It's not needed